app/models/vote.rb

```diff
@@ -0,0 +1,6 @@
+class Vote < ApplicationRecord
+  belongs_to :user
+  belongs_to :work
```

Comment:
You can't delete a work that has votes! This is because you've created a link between the two tables at the database level, and deleting the work would leave behind invalid votes. You might want to check out the [dependent](http://guides.rubyonrails.org/association_basics.html#dependent) argument to `has_many`.